salt/states/tls.py

```diff
@@ -38,7 +38,12 @@ def valid_certificate(
            'comment': ''}
 
     now = time.time()
-    cert_info = __salt__['tls.cert_info'](name)
+    try:
+        cert_info = __salt__['tls.cert_info'](name)
+    except IOError as exc:
```

Comment:
IMHO, we should at least add a `log.error` message here.

Reply:
Like: https://github.com/saltstack/salt/pull/46617/commits/48b73f46cbfae777f2ae2a4d7c31ed423343064d ?